stimela/kitchen/cab.py

```diff
@@ -313,12 +315,19 @@ def stringify_argument(name, value, schema, option=None):
             if schema.dtype == "bool":
                 if key_value:
                     args += [f"{name}={value}"]
```

Comment:
Spotted an old bug here -- ``{name}`` should be ``{schema.nom_de_guerre or name}``.

Reply:
> should we not rather have key_value: true to represent this case (and empty prefix, key_value: true for the option=value case)? 

That's what I tried at first, but it doesn't work since `key_val` policy does not treat lists and `explicit_{true|false}` cases. I'm assuming `key_val` was not really meant for a command-line parser.

Reply:
> I'm assuming key_val was not really meant for a command-line parser.

But it was -- there are tools (e.g. QuartiCal) that use key=value command lines. So any problems should be treated as bug to be fixed. ``explicit_{true|false}`` was meant to work with it. As for lists, ``repeat_policy=list`` makes no sense with this construct as far as I can see (see other remark in this review), but other repeat_policies should work.

Reply:
Noted, I'll add expand the functionality of `key_val`. Making this a draft PR for now, will re-open later